interfaces/cython/cantera/kinetics.pyx

```diff
@@ -366,6 +366,12 @@ cdef class InterfaceKinetics(Kinetics):
         coverages for a specified amount of time.
         """
         (<CxxInterfaceKinetics*>self.kinetics).advanceCoverages(dt)
+      
+    def advance_coverages_to_steady_state(self):
+        """
+        This method advances the surface coverages to (pseudo) steady state.
```

Comment:
I know this comes from the name of the underlying method, but can you either explain what "pseudo" means here, or remove it from the description, depending on what you think makes more sense?

Reply:
For me it is OK just to remove it from the description.

Reply:
Can you push a revised version of the PR, then?